corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -990,7 +1000,7 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                             return false;
                         },
                     },
-                }, caseConfig);
+                }, caseConfig, privileges.subcases);
```

Comment:
@sravfeyn to answer your question from https://github.com/dimagi/commcare-hq/pull/25087#pullrequestreview-275369840 this is where it can be set to `false`